python/graphframes/examples.py

```diff
@@ -57,3 +60,216 @@ def friends(self):
         ], ["src", "dst", "relationship"])
         # Create a GraphFrame
         return GraphFrame(v, e)
+
+    def gridIsingModel(self, n, vStd=1.0, eStd=1.0):
+        """Grid Ising model with random parameters.
```

Comment:
I also think this module should be split into subpackage `examples` with modules `Graphs` and `BeliefPropagation`, just as in Scala side. I just did not have courage to do it without clarification from you. I'll do it.